### DISCUSSION TITLE
Colors on my code

### DISCUSSION THREAD

**Discussion by Abdullah-Mahthel** [helix-editor/helix#7413] [Q&A]
![Helixpng](https://github.com/helix-editor/helix/assets/130181169/e023435f-1a71-4997-ab69-f41d0f0ba53e)

please tell me why there is no colors on my code like :

data type with unique color, variables ..etc 

How can l change that ?
Is there a configuration?

**Comment by gabydd** [comment]
hard to tell exactly why, some suggestions:
- to know what to colour your code helix needs a file extension to know what language it is or you can use ``:set-language <lang>`` to manually set the buffers language
- you should check ``hx --health`` to make sure you have all the runtime files helix needs for highlighting, depending how you installed helix things can be different so if you give some more details about how you installed helix that could be helpful

**Reply by Abdullah-Mahthel** [reply]
![config](https://github.com/helix-editor/helix/assets/130181169/f915ecbd-ab3a-474d-b0ce-1ce9fffe37b6)

How can l save language?
When l close and open helix again . it doesn't recognize cpp again :/
